lib/rust/ensogl/component/text/src/typeface/glsl/glyph.glsl

```diff
@@ -1,7 +1,11 @@
+// A factor describing much the bold letters will be fattened, expressed as the fraction of font size.
+const float BOLD_FATTING = 0.04;
+
 highp float median(highp vec3 v) {
     return max(min(v.x, v.y), min(max(v.x, v.y), v.z));
 }
 
+// To smaple the msdf properly, the boundaries of our uv should be in the middle of msdf cells.
```

Comment:
smaple

Reply:
Plus, I dont understand this comment – what do you mean by "boundaries" to be in the middle of msdf cells? Could you explain it better, please?

Reply:
I changed the name and comment. Would you like to check if it's better?